news_items/008_honor_code/index.html

```diff
@@ -0,0 +1,43 @@
+
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+</style>
+
+<p>
+  Thanks to <a href="https://github.com/tdy">Tim Yang</a>,
+  PrairieLearn exams now default to starting with an identity
+  certification and honor code pledge:
+</p>
+
+<img src="honor_code.png" width="700px">
+
+<p>
+  The "Start assessment" button is disabled until the student clicks
+  the checkbox to "certify and pledge the above".
```

Comment:
```suggestion
  the checkbox to "certify and pledge the above."
```
Unless it's an exception? I can never tell 😩

Reply:
It's an American/British thing. Americans always put them inside, while in British English things are more flexible. I personally can't stand the logical inconsistency of putting something inside the quotes that isn't really part of the quotation (as demanded by American English), but in this case we are fortunate that the period is part of the quotation so inside-the-quotes will be correct under either tradition!